app/Controllers/LoanApprovalController.php

```
@@ -192,7 +192,67 @@ public function approveLoan(Request $request, Response $response, mysqli $db): R
+                $response->getBody()->write(json_encode([
+                    'success' => false,
+                    'message' => __('L\'utente ha già un prestito attivo per questo libro')
+                ]));
+                return $response->withHeader('Content-Type', 'application/json')->withStatus(409);
+            }
+
+            // Enforce max_active_loans_per_user anche in approvazione (P1): il limite era
+            // applicato solo alla creazione/richiesta, ma approvando più richieste pendenti
+            // su libri diversi un admin poteva superarlo. Conta i prestiti attivi escluso
+            // quello in approvazione; il lock sulla riga libro sopra serializza.
+            $maxLoans = (int) ((new \App\Models\SettingsRepository($db))->get('loans', 'max_active_loans_per_user', '0') ?? 0);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '240', column '37'). (undefined)

(MissingImport)